docs/glossary.md

```diff
@@ -28,6 +28,17 @@ Cookiecutter
     A command-line utility that creates projects from cookiecutters (project templates), for example, creating a Python package project from a Python package project template.
     [See Cookiecutter's documentation](https://cookiecutter.readthedocs.io/en/stable/).
 
+Cookieplone
+    ```{versionadded} Volto 18.0.0-alpha.43
+    ```
+
+    [Cookieplone](https://github.com/plone/cookieplone) is the method to create a Plone project.
+    You can use Cookieplone to build a backend add-on, a new Volto add-on, a full project with both backend and frontend, or even documentation.
```

Comment:
```suggestion
    You can use Cookieplone to build a backend add-on, a new Volto add-on, or a full project with both backend and frontend.
```

(I think adding a documentation template is a great idea, but it doesn't exist yet.)

Reply:
I pulled that from its readme. See https://github.com/plone/cookieplone/pull/40